src/libs/mps_comm/opcua/machine.cpp

```diff
@@ -170,7 +170,9 @@ OpcUaMachine::connect()
 	if (connection_mode_ == MOCKUP) {
 		return;
 	}
-	while (!reconnect()) {}
+	while (!reconnect()) {
+		std::this_thread::sleep_for(std::chrono::milliseconds(40));
```

Comment:
Maybe this should be configurable, a variable, or a macro as it is used multiple times.

Reply:
i changed that to be a inline constexpr, good catch!